integrations/vello_svg/src/lib.rs

```diff
@@ -33,78 +33,107 @@
 //! - patterns
 
 use std::convert::Infallible;
-
 use usvg::NodeExt;
-use vello::kurbo::{Affine, BezPath, Rect, Stroke};
-use vello::peniko::{Brush, Color, Fill};
-use vello::SceneBuilder;
+use vello::{
+    kurbo::{Affine, BezPath, Point, Rect, Stroke},
+    peniko::{Brush, Color, Fill},
+    SceneBuilder,
+};
+
+/// Re-export vello.
+pub use vello;
 
+/// Re-export usvg.
 pub use usvg;
 
-/// Append a [`usvg::Tree`] into a Vello [`SceneBuilder`], with default error handling
-/// This will draw a red box over (some) unsupported elements
+/// Append a [`usvg::Tree`] into a Vello [`SceneBuilder`], with default error
+/// handling This will draw a red box over (some) unsupported elements
 ///
 /// Calls [`render_tree_with`] with an error handler implementing the above.
 ///
-/// See the [module level documentation](crate#unsupported-features) for a list of some unsupported svg features
-pub fn render_tree(sb: &mut SceneBuilder, svg: &usvg::Tree) {
-    render_tree_with(sb, svg, default_error_handler).unwrap_or_else(|e| match e {});
+/// See the [module level documentation](crate#unsupported-features) for a list
+/// of some unsupported svg features
+pub fn render_tree(sb: &mut SceneBuilder, svg: &usvg::Tree, transform: Option<Affine>) {
```

Comment:
Why is this transform needed? You can append a fragment created using this to a scene with an applied transform